.pre-commit-config.yaml

```
@@ -8,3 +8,7 @@ repos:
     rev: v5.5.3
     hooks:
       - id: isort
+-   repo: https://gitlab.com/pycqa/flake8
+    rev: 3.7.9
+    hooks:
+    - id: flake8
```

```suggestion
    - id: flake8
      args: ["--ignore=E501,W"]
```

We need to ignore the same parameters as the github action ignores.